packages/codecs/package.json

```diff
@@ -0,0 +1,94 @@
+{
+    "name": "@solana/codecs",
+    "version": "2.0.0-development",
+    "description": "Helpers for creating transaction codecs",
```

Comment:
Nit: should this be more generic?

Reply:
Shoot good catch! I actually forgot to update this field. This comes from copy/pasting the `instructions` package and `sed s/instructions/codecs/g`. 😬 

I'll update now.